config/environments/production.rb

```diff
@@ -60,6 +60,7 @@
 
   # Use a different cache store in production.
   # config.cache_store = :mem_cache_store
+  config.cache_store = :memory_store, { size: 10.megabytes }
```

Comment:
This was just for performance testing. For production we'll use a memcached/redis instance on aws.